README.md

```diff
@@ -218,6 +218,30 @@ app({ model, view, update })
 [View online](http://codepen.io/jbucaran/pen/zNxRLy?editors=0010)
 </details>
 
+<details>
+<summary>JSX</summary>
+
+```jsx
+const view = (model, msg) => (
+    <div>
+        <button onclick={msg.add}>Add</button>
+        <h1>{model}</h1>
+        <button onclick={msg.sub}>Subtract</button>
```

Comment:
The original example has a disabled html property that turns to true when the number is <= 0

Awesome work though!